.github/workflows/checks.yml

```diff
@@ -67,9 +67,10 @@ jobs:
       - name: Typechecking
         run: |
           poetry run poe typecheck
-      - name: Check if requirements.txt is up-to-date
+      - name: Check if requirements.txt and docs-requirements.txt are up-to-date
         run: |
-          poetry export -f requirements.txt --without-hashes --extras docs | cmp - requirements.txt
+          poetry export -f requirements.txt --without-hashes | cmp - requirements.txt
+          poetry export -f requirements.txt --without-hashes --extras docs | cmp - docs-requirements.txt
```

Comment:
Is `docs-requirements` okay? Isn't `optional-requirements` kind of conventional?

Reply:
In our case, those requirements are only for the docs, but to be more future proof I can change the name to `optional`